GitUI/Editor/FileViewer.cs

```diff
@@ -530,36 +537,42 @@ public void ViewCurrentChanges(GitItemStatus item, bool isStaged, [CanBeNull] Ac
                 });
         }
 
-        public void ViewStagingPatch(Patch patch, [CanBeNull] Action openWithDifftool)
+        public void ViewStagingPatch(Patch patch, [CanBeNull] Action openWithDifftool, string filename)
         {
-            ViewPatch(patch, openWithDifftool);
+            ViewPatch(patch, openWithDifftool, filename);
             Reset(true, true, true);
         }
 
-        public void ViewPatch([CanBeNull] Patch patch, [CanBeNull] Action openWithDifftool = null)
+        public void ViewPatch([CanBeNull] Patch patch, [CanBeNull] Action openWithDifftool = null, string filename = null)
         {
-            ViewPatch(patch?.Text ?? "", openWithDifftool);
+            ViewPatch(patch?.Text ?? "", openWithDifftool, filename);
         }
 
-        public void ViewPatch([NotNull] string text, [CanBeNull] Action openWithDifftool)
+        public void ViewPatch([NotNull] string text, [CanBeNull] Action openWithDifftool, string filename = null)
```

Comment:
I guess, this function is always called with all arguments, i.e. the default argument value can be omitted.
```suggestion
        public void ViewPatch([NotNull] string text, [CanBeNull] Action openWithDifftool, [CanBeNull] string filename)
```